docs/_data/examples.yml

```diff
@@ -0,0 +1,146 @@
+- id: terraform-http-basic
+  name: HTTP Basic
+  image: /assets/img/logos/terraform-logo.png
+  files:
+    - name: main.tf

```

Comment:
Oh, one more thought: couldn't we determine the name automatically from the URL? That is, it's just the text after the final slash, right? Would reduce redundancy in this code.